implementations/rust/ockam/ockam_api/src/cli_state/nodes.rs

```diff
@@ -223,55 +233,61 @@ impl CliState {
         pid: u32,
         signal: signal::Signal,
     ) -> Result<()> {
+        debug!(%pid, %signal, "sending kill signals to node's process");
         let pid = nix::unistd::Pid::from_raw(pid as i32);
-        signal::kill(pid, signal)
-            .or_else(|e| {
-                if e == Errno::ESRCH {
-                    tracing::warn!(node = %node_name, %pid, "No such process");
-                    Ok(())
-                } else {
-                    Err(e)
-                }
-            })
-            .map_err(|e| {
-                CliStateError::Io(std::io::Error::new(
-                    std::io::ErrorKind::Other,
-                    format!("failed to stop PID `{pid}` with error `{e}`"),
-                ))
-            })?;
-        debug!(name = %node_name, %pid, %signal, "sent stop signal to node's process");
+        let _ = self.send_kill_signal(pid, signal);
 
         // Wait until the node has fully stopped
         let timeout = std::time::Duration::from_millis(100);
-        let max_attempts = std::time::Duration::from_secs(5).as_millis() / timeout.as_millis();
+        let max_attempts = std::time::Duration::from_secs(3).as_millis() / timeout.as_millis();
```

Comment:
Could we please leave it as 5 seconds? That's pretty long, but on VMs eBPF portals sometimes take some time to shut down, sometimes even more than 5 seconds. But I would make it 5 seconds and then I will investigate why it's so long

Reply:
Yes sorry, I forgot to roll that back